src/beaverdam/viewer/presenter.py

```diff
@@ -109,3 +109,18 @@ def get_element_properties(self, element_id):
         element_properties = all_elements[element_id]["properties"]
 
         return element_properties
+
+    def get_element_contents(self, element_id):
+        """Get the contents of an element, given the element's id.
+
+        Args:
+            element_id (str):  the unique id of the element
+
+        Returns:
+            element_contents (dict):  properties of the element
+
+        """
+        all_elements = self.get_elements()
+        element_contents = all_elements[element_id]["contents"]
```

Comment:
Could be concatenate
```suggestion
        element_contents = self.get_elements()[element_id]["contents"]
```